bottlerocket/testsys/src/run_aws_ecs.rs

```diff
@@ -146,6 +146,10 @@ pub(crate) struct RunAwsEcs {
     /// The arn for the role that should be assumed by the agents.
     #[structopt(long)]
     assume_role: Option<String>,
+
+    /// The IAM instance profile name. If no value is provided the default name will be used.
```

Comment:
I think we can elaborate a bit more on this.
```suggestion
    /// The IAM instance profile name for the EC2 instances in the ECS cluster. If no value is provided, then the ECS test agent will attempt to create an IAM instance profile.
```